newsService/src/test/java/com/kovan/NewsServiceTest.java

```diff
@@ -35,6 +35,9 @@ class NewsServiceTest {
     @Mock
     private NewsRepository newsRepository;
 
+    @Mock
+    private NewsEntity newsEntity;
```

Comment:
why do we need this? give me an explanation.

Reply:
Removed @mock for entity.